src/k8s/pkg/utils/time.go

```diff
@@ -0,0 +1,78 @@
+package utils
```

Comment:
@mateoflorido i think this package is what i was hoping for.  I like the approximations.